chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/TestFunctionalityEntity.java

```diff
@@ -36,6 +36,10 @@
     @JoinColumn(name = "certification_edition_id", insertable = false, updatable = false)
     private CertificationEditionEntity certificationEdition;
 
+    @OneToOne(optional = true, fetch = FetchType.EAGER)
```

Comment:
I don't think I've seen "EAGER" before. Most of the others use "LAZY". What's the difference? Why one or the other?

Reply:
I usually do Lazy fetches and wherever the test functionality entity is being gotten in the dao i'll add "join fetch" code to also get the practiceType entity. I don't like hibernate doing extra queries behind the scenes which I think is what Eager fetches have the potential to do?